install_config/persistent_storage/topics/glusterfs_prereqs_hardware.adoc

```diff
@@ -0,0 +1,30 @@
+Any nodes used in a {gluster-native} or {gluster-external} cluster are
+considered storage nodes. Storage nodes can be grouped into distinct cluster
+groups though a single node may not be in multiple groups. For each group of
```

Comment:
s/groups though/groups, though

Also, "may not" is confusing here. Does the single node have the ability to be in multiple groups, but is not, only b/c of configuration, rules, etc.? In most instances, we change "may" to “can” or “might."